### DISCUSSION TITLE
Multiple keys with same meaning i.e. same translation?

### DISCUSSION THREAD

**Discussion by Prochy** [WeblateOrg/weblate#11651] [Q&A]
I tried to search for similar question but I didn't find anything. Is it possible with Weblate have multiple keys that have the same meaning and it one is translated it will translate also the others cross reference keys? 

Lets say I have C# app localized through x:Uid where I have a button with x:Uid="ButtonToolTip", in the resource file the key is "Button.ToolTipService.ToolTip" = "This is the localized tooltip.".

 Then I have Swift application where the storyboard is localized in a different way and I have there string "ButtonToolTip":"This is the localized tooltip.".

I would like to tell Weblate that the key "Button.ToolTipService.ToolTip" is  same as "ButtonToolTip" so when someone translate Button.ToolTipService.Tooltip it will automatically affect "ButtonToolTip" as well. 

Not sure if this is supported or not.

**Comment by Akemi-Madoka** [comment]
I am not certain whether the approach you're describing can be utilized here, but it can serve as a reference for you:

In our translation project, in most cases, the same source text is translated into the same target text, but in a few cases, it should be translated differently. Therefore, we have traversed all documents that need translation and set their locations as keys. The source text is imported as a monolingual file, and then a target language translation file is created.

At this point, we add an "auto translation" component within the component, setting it to translate according to all components in the project.

Finally, we translate the repeated source text once in the project, and then wait. Based on our experience, our project will start auto-translation around 13:00 each day, and the component will automatically translate entries with the same source text and untranslated into the same entries that have already been translated.
